subprojects/docs/docs.gradle

```diff
@@ -587,7 +587,6 @@ tasks.named("docsTest") { task ->
             excludeTestsMatching 'org.gradle.docs.samples.ExemplarExternalSamplesFunctionalTest.snippet-testing-jacoco-application_*_jacocoApp.sample'
         }
 
-        excludeTestsMatching 'org.gradle.docs.samples.ExemplarExternalSamplesFunctionalTest.android-application_*.sample' // has build deprecations
```

Comment:
👍 Thanks for taking the time to fix the sample and the corresponding test coverage.